cogs/info.py

```diff
@@ -107,6 +107,17 @@ async def about(self, ctx):
 
         await ctx.send(content=f"ℹ About **{ctx.bot.user}** | **{self.config['version']}**", embed=embed)
 
+    ## Logging messages sent in chat for use with @story{11}
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        log_file = open("LogFile.txt", "a")
+        msg = """
+User:{}
```

Comment:
Formatting here may be getting messed up because of github but you may want to double check your editor to ensure it is using tabs and spaces correctly and that they are set to the proper widths